package.json

```diff
@@ -18,6 +18,7 @@
   "author": "Hugh Francis",
   "license": "MIT",
   "devDependencies": {
+    "ember-data": "1.13.11",
```

Comment:
Hey! I thought so too at first, but I tried removing it with no luck. Almost all tests are failing when ember-data is removed from package.json.